rust/core-lib/src/editor.rs

```diff
@@ -311,27 +209,10 @@ impl Editor {
         self.gc_undos();
     }
 
-    /// Returns buffer information used to initialize plugins.
-    pub fn plugin_init_info(&self) -> PluginBufferInfo {
-        let nb_lines = self.text.measure::<LinesMetric>() + 1;
-        let views = vec![self.view.view_id];
-        let config = self.config.to_table();
-        PluginBufferInfo::new(self.buffer_id, &views,
-                              self.engine.get_head_rev_id().token(), self.text.len(),
-                              nb_lines, self.path.clone(), self.syntax.clone(),
-                              config)
-    }
-
-    /// Send initial config state to the client.
-    pub fn send_config_init(&self) {
-        let config = self.config.to_table();
-        self.doc_ctx.config_changed(&self.view.view_id, &config);
-    }
-
-    fn insert(&mut self, s: &str) {
+    fn insert(&mut self, view: &View, s: &str) {
```

Comment:
This would be another candidate for an `Into<Rope>` arg, but less important here.